src/main/resources/application.yml

```diff
@@ -38,6 +38,12 @@ spring:
           jwk-set-uri: ${api.base.url.hmpps-auth}/.well-known/jwks.json
       client:
         registration:
+          activities-appointments:
+            provider: hmpps-auth
+            client-id: ${system.client.id}
+            client-secret: ${system.client.secret}
+            authorization-grant-type: client_credentials
+            scope: read
```

Comment:
Probably will need to be read/write here - but fine for now.

Reply:
Good spot.